### ISSUE TITLE
I managed to kill the server again.

### ISSUE THREAD

**Issue by nullbear** [Apollo-Community/ApolloStation#172]
Using the desub chamber, i made a new supermatter. I ran into the supermatter and died.

Quite some time later, Grant tossed a bunch of things, like 300 things at once (air pressure, it wasnt intentional, THE FOOL!) into the supermatter, triggering its delamination. He stuck it on the cargo shuttle, saying something like "its centcomms problem now" but wasnt able to send the cargo shuttle away.

When the supermatter blew up on the cargo shuttle, connection died, and could no longer reconnect to the server.

**Comment by Kwask** [comment]
I suspect it may be from a certain turf getting deleted, but I'll have to check the logs

**Comment by lmas** [comment]
Now that's just friggin awesome :D Time to start experimenting with spare SM cores.

**Comment by lmas** [comment]
So I tried [detonating the SM on the shuttle too, no crash. I then tried do a manual explosion](http://imgur.com/KofqVKO) (30,30,30,30 something) and got a bunch of runtimes from a mulebot and the mouse nearby. No server crashes though.

Edit:
Tried it another time and sending away the shuttle. No crash. And the shuttle jumped away with the SM still on it.

Edit2: I used the SM core inside the engine room, not one grown by hand in the desub lab.
